### ISSUE TITLE
Greaseweazle v4 and brother format

### ISSUE THREAD

**Issue by ZHolyhell** [davidgiven/fluxengine#440]
The greaseweazle v4 does not appear to want to write the brother 120 or 240 formats.

It writes and verifys 6 times then posts error: write failed: uncorrectable error during write.

**Comment by davidgiven** [comment]
Are you using an HD or DD disk? You can't write Brother formats to HD disks (the pulse intervals are too wide). I can verify here that I can write DD disks but not HD disks, which produce the symptoms you describe.

**Comment by ZHolyhell** [comment]
It appears you are correct it was the disk (none of my disks have density
labels and they came with the word processor) but what was written when I
found a disk that wrote the image had 3 corrupted files (including the
spreadsheet itself) though the original works fine.  I will try writing
again next.

**Comment by davidgiven** [comment]
HD disks have two holes at the top; DD disks only have one.

Did I understand you correctly that you imaged the original disk, wrote it back to another disk, and got corrupt files? That shouldn't happen...

**Comment by ZHolyhell** [comment]
Here is an image that will produce a working version of the spreadsheet disk

**Comment by ZHolyhell** [comment]
Yes I did have that happen it appears that sometimes it does not completely read the diskette and will sometimes miss a spot in a write repeating a few times over the same disks worked out though 

I wound up having to read 3 times to get an image that had the most bits on docs contained within  and write that image to another disk twice to get it to produce a successful image on it

**Comment by davidgiven** [comment]
If it fails partway through, try increasing the number of retries --- `--decoder.retries=10` or so.

**Comment by ZHolyhell** [comment]
It never told me it failed it just wasn't 100% the first time reading or writing 
I would recommend it compare file created/read from to disk after creation though it would take longer to complete overall it would be a little  more user friendly overall and ensure accuracy of both

**Comment by ZHolyhell** [comment]
Well I sent all I could to your cowlark email since github didn't seem to like the files

**Comment by davidgiven** [comment]
I have actually obtained a WP-1, and discovered that the brother120 encoder was horribly broken. I've fixed it. Could you upgrade to the latest development build and try again, please?

**Comment by ZHolyhell** [comment]
Uh my brother wp-2410 may use .wp1 word processor files but it uses the brother240 disk format

**Comment by davidgiven** [comment]
You said you were writing 120kB disks?

**Comment by ZHolyhell** [comment]
I was testing to see if it could since the 240 wasn't working we solved that part this is still open for the fragmentation/corruption issue
